sphinx/writers/html.py

```diff
@@ -874,11 +874,29 @@ def visit_field_name(self, node):
 
     def visit_math(self, node, math_env=''):
         # type: (nodes.Node, unicode) -> None
-        logger.warning(__('using "math" markup without a Sphinx math extension '
-                          'active, please use one of the math extensions '
-                          'described at http://sphinx-doc.org/en/master/ext/math.html'),
-                       location=(self.builder.current_docname, node.line))
-        raise nodes.SkipNode
+        name = self.builder.math_renderer
+        visit, depart = self.builder.app.registry.html_inline_math_renderers[name]
```

Comment:
`visit, _ = self.builder....` is better because `depart` is never used.

Reply:
done